.gitignore

```diff
@@ -4,4 +4,5 @@
 .idea
 /docs/site
 bin
-build
\ No newline at end of file
+build
+/deploymentFiles
```

Comment:
are these specific to ACK

Reply:
no that's just a directory i created, will remove